migration/build-test-runner.md

```diff
@@ -0,0 +1,13 @@
+# Build Test Runner
+
+In Exercism V3, one of the biggest changes is that we'll automatically check if a submitted solution passes all the tests.
+
+We'll check this via a new, track-specific tool: the [_test runner_](https://github.com/exercism/v3-docs/tree/master/anatomy/track-tooling/test-runners). The test runner is track-specific and implemented as a Docker image. Whenever a solution is submitted, a Docker container is created of that image and it runs on the submitted solution. The test runner outputs a JSON file that describes the test results.

```

Comment:
I worry a little that we scare people by mentioning Docker here. The Docker bit is actually a small/easy bit that we can help with, but for lots of people it might be offputting.

```suggestion
We'll check this via a new, track-specific tool: the [_test runner_](https://github.com/exercism/v3-docs/tree/master/anatomy/track-tooling/test-runners). Each test runner is track-specific. When a new solution is submitted, we run the track's test-runner, which outputs a JSON file that describes the test results.

```

Reply:
Fair point